### PR TITLE
did dock

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

**Comment by osinakayah** [comment]
@maycon-mello 
Do we need a function for calculating the transaction fee needed for create a did:dock?

File: packages/core/lib/services/dids/index.test.js [hunk 1]
```diff
@@ -99,6 +99,6 @@ describe('DID Service', () => {
 
   it('expect to register did dock', async () => {
```
Comment (osinakayah) [inline packages/core/lib/services/dids/index.test.js:100]: Can we include another test case for when something goes wrong.
Like, if no Keypair was found, or if there is insufficient for the transaction fee.

**Review by osinakayah — CHANGES_REQUESTED** [review]

**Comment by maycon-mello** [comment]
> @maycon-mello Do we need a function for calculating the transaction fee needed for create a did:dock?

I can add one in a separate PR

**Comment by osinakayah** [comment]
Some tests are failing.

**Comment by maycon-mello** [comment]
The failed tests seems related to the substrate mock, I'm working to fix that. Also, its showing as passing in the CI again, but shouldn't

**Review by osinakayah — APPROVED** [review]

**Pull request merged**